desktop-widgets/filterwidget.ui

```diff
@@ -88,7 +88,7 @@
        </property>
        <property name="icon">
         <iconset resource="../subsurface.qrc">
-         <normaloff>:filter-hide</normaloff>:filter-hide</iconset>
+         <normaloff>:show-icon</normaloff>:show-icon</iconset>
```

Comment:
This one surprised me. I haven't looked at the icons - just the switch from 'hide' to 'show'...

Reply:
Current name is misleading, because the same icon is used for show and hide actions. So this doesn't matter too much. Hopefully, we will use different icons someday. I will change this to *hide*.